configuration/pih/liquibase/sql/create_functions.sql

```diff
@@ -72,8 +72,9 @@ BEGIN
 	WHERE voided = 0
 	  AND concept_id = _conceptID
 	  AND locale = _locale
- 	order by locale_preferred desc, ISNULL(concept_name_type) asc, 
-	field(concept_name_type,'FULLY_SPECIFIED','SHORT')
+	order by if(_locale = locale, 0, 1), if(locale = 'en', 0, 1),
+	  locale_preferred desc, ISNULL(concept_name_type) asc, 
+	  field(concept_name_type,'FULLY_SPECIFIED','SHORT')
```

Comment:
This is a cool function - I didn't know about it.  So this works with the ISNULL above to return FULLY SPECIFIED first, SHORT second, and then anything else after (assuming ordering is already the same).